packages/zapp/console/src/components/Launch/LaunchForm/ResumeForm.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { createInputValueCache, InputValueCacheContext } from './inputValueCache';
+import { ResumeSignalForm } from './ResumeSignalForm';
+import { ResumeSignalFormProps } from './types';
+
+/** Renders the form for initiating a Launch request based on a Workflow or Task */
+export const ResumeForm: React.FC<ResumeSignalFormProps> = (props) => {
+  const [inputValueCache] = React.useState(createInputValueCache());
```

Comment:
update hook import

Reply:
Ok, I don't think you use `inputValueCache` anywhere in the resume form, so we don't need this wrapper - `<ResumeSignalForm />` can be used directly in its place

Reply:
even better would be to rename `ResumeSignalForm` as `ResumeForm` and delete this component